spec/features/groups/manage_group_members_spec.rb

```diff
@@ -50,19 +64,55 @@ def when_i_click_add_editor
     click_link "Add an editor"
   end
 
+  def when_i_click_add_editor_or_group_admin
+    click_link "Add an editor or group admin"
+  end
+
   def then_i_should_see_the_add_editor_form
     expect(page.find("h1")).to have_text "Add an editor to this group"
     expect_page_to_have_no_axe_errors(page)
   end
 
+  def then_i_should_see_the_add_editor_or_group_admin_form
+    expect(page.find("h1")).to have_text "Add an editor or group admin to this group"
+    expect_page_to_have_no_axe_errors(page)
+  end
+
   def when_i_fill_in_the_add_editor_form
     fill_in "Enter the email address of the person you want to add", with: user3.email
     click_button "Add this person"
   end
 
-  def then_i_should_see_the_new_editor_in_the_list
+  def when_i_fill_in_the_add_group_admin_form
+    fill_in "Enter the email address of the person you want to add", with: user3.email
+    choose("Group admin")
+    click_button "Add this person"
+  end
+
+  def then_i_should_see_the_new_group_admin
+    expect(page.find("h1")).to have_text "Group 1"
+
+    user_row = page.find("td", text: user3.name).ancestor("tr")
```

Comment:
I see this is used a few times in this file, it might be worth putting this into a `let`  or perhaps a private method 🤔